mad-dog/src/index.js

```diff
@@ -17,6 +18,33 @@ const DEFAULT_NUM_USERS = 2
 const SNAPBACK_NUM_USERS = 10
 const USER_REPLICA_SET_NUM_USERS = 4
 
+const SERVICES = [
```

Comment:
why is this necessary?

Reply:
this was so that we can easily perform a health check on all the local services